packages/langium/src/workspace/documents.ts

```diff
@@ -95,10 +94,15 @@ export enum DocumentState {
 }
 
 /**
- * Result of the scope precomputation phase (`ScopeComputation` service).
- * It maps every AST node to the set of symbols that are visible in the subtree of that node.
+ * Result of the scope pre-computation phase performed by the `ScopeComputation` service.
+ * It maps AST nodes of a document to their corresponding sets of symbols that are accessible
+ * by those nodes/subtrees, provided any symbols corresponding specifically to those nodes/subtrees exist.
+ * The sets of symbols are assumed to be un-ordered. Hence, no assumptions about the order of symbols in the sets should be made.
```

Comment:
I'd add that usually a MultiMap is used for collecting symbols, which conforms to this interface.